service/controller/v24/resource/cloudformation/delete.go

```diff
@@ -100,6 +108,47 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 	return nil
 }
 
+func disableMasterTerminationProtection(ctx context.Context, masterInstanceName string) error {
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	input := &ec2.DescribeInstancesInput{
+		Filters: []*ec2.Filter{
+			{
+				Name: aws.String("tag:Name"),
+				Values: []*string{
+					aws.String(masterInstanceName),
+				},
+			},
+		},
+	}
+	result, err := cc.AWSClient.EC2.DescribeInstances(input)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	for _, reservation := range result.Reservations {
```

Comment:
For now we only have one master and we should verify we only modify one. When there are more masters in the result, or fewer, we should return an error.